testsuite/rtest

```diff
@@ -9,22 +9,32 @@ use File::Temp qw/ tempfile /;
 (undef,$testTempFile) = tempfile("rtest$$.XXXXX", TMPDIR => 1, SUFFIX => ".tmp", UNLINK => 1);
 $cwd = getcwd;
 if ($cwd =~ m/(.*)testsuite\/(.+)$/) {
+  $PLATFORM1 = $ENV{'PLATFORM1'};
+  $PLATFORM2 = $ENV{'PLATFORM2'};
+  $PLATFORM3 = $ENV{'PLATFORM3'};
```

Comment:
Why are you not using `-platform=mingw` to override `$PLATFORM2` and `$PLATFORM3`?

Reply:
There wasn't such an option :D